web/locales/de.json

```diff
@@ -1495,5 +1496,25 @@
         "map-info-text": "Diese Karte veranschaulicht in Echtzeit, wie viel CO2-Äquivalente bei der Stromerzeugung freigesetzt werden. Die Färbung der Zonen erfolgt anhand des spezifischen Ausstoßes von Treibhausgasen (CO2-Intensität), die durch die Deckung des Strombedarfs einer Region emittiert werden. Eine grüne Färbung steht dabei für eine klimafreundlichere Erzeugung.",
         "electricity-origins-text": "Die Pfeile auf der Karte stellen den physischen Stromaustausch zwischen Regionen dar. Tippe auf eine Region, um mehr Details zur Herkunft des verfügbaren Stromes zu erhalten.",
         "wind-solar-text": "Die Verfügbarkeit von Wind- und Solarstrom ist wetterabhängig. Tippe auf den Wind- und Sonnenbutton, um das globale Echtzeit-Potenzial dieser Ressourcen anzuzeigen."
+    },
+    "localeNames": {
+        "AR": "Arabisch",
+        "CS": "Tschechisch",
+        "DA": "Dänisch",
+        "DE": "Deutsch",
+        "EN": "Englisch",
+        "ES": "Spanisch",
+        "FR": "Französisch",
+        "IT": "Italienisch",
+        "JA": "Janpanisch",
+        "NL": "Niederlöändisch",
+        "PL": "Polnisch",
+        "PT-BR": "Brazilianisches Portugiesisch",
+        "RU": "Russisch",
+        "SK": "Slowenisch",
+        "SV": "Schwedisch",
+        "ZH-CN": "Mainland China Chinesisch",
```

Comment:
I think in German this could be called "Festlandchina", see https://de.wikipedia.org/wiki/Festlandchina

And how about phrasing these as "Chinesisch (Festlandchina)", "Chinesisch (Taiwan)", "Chinesisch (Hong Kong)?

Reply:
We should have all languages in the language of the reader. If I speak French, it is more easy to select "Français" than "Französisch" ;)  
Something like :
Deutsch,
English,
Français
中文

Reply:
And providing only the English translation if we don't have the language name in the langage. In that case, my hypothesis : there is more chance that a reader know the name of his language in English than German.

Reply:
@jarek I Agree with you